base/docs/helpdb/Base.jl

```diff
@@ -8719,29 +8719,14 @@ alias the input `x` to modify it in-place.
 filt!
 
 """
-    ascii(::Array{UInt8,1})
+    ascii(s::AbstractString)
```

Comment:
This is what the script uses to find the correct place to insert the docstring so changing the signature will make the doc generation fail.